### PR TITLE
RNs-4.8.46:adds zstream update

### PR DESCRIPTION
**Pull request by JoeAldinger**
RNs-4.8.46: adds zstream and fixes typo in RNs

Version(s):
4.8

Issue:

Link to docs preview:
Preview for [upgrading](https://joealdinger.github.io/openshift-docs/RNs-4.8.46/release_notes/ocp-4-8-release-notes.html)
Preview for [RNs 4.8.46](https://joealdinger.github.io/openshift-docs/RNs-4.8.46/release_notes/ocp-4-8-release-notes.html#ocp-4-8-46)

Additional information:
Along with the zstream update to RNs, I also fixed a typo changing updating to upgrading.

### CODE REVIEW

**Comment by aireilly** [comment]
LGTM

**Review by aireilly — APPROVED** [review]

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -3397,7 +3397,22 @@ link:https://access.redhat.com/solutions/6965038[{product-title} 4.8.45 containe
...
+
+{product-title} release 4.8.46 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5424[RHBA-2022:5424] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5423[RHBA-2022:5423] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6966480[{product-title} 4.8.46 container image list]
+
+==== Updating
 
-To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (opayne1) [inline release_notes/ocp-4-8-release-notes.adoc:3403]: Is there a reason you removed "within a minor version by"?

**Review by opayne1 — COMMENTED** [review]
2 little things

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-8-release-notes.adoc:3403]: nvm i see lol

Comment (JoeAldinger) [reply release_notes/ocp-4-8-release-notes.adoc:3403]: Appreciate the update 😁. Sorry, I had to 🤣.

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**